include/sound/sof/header.h

```diff
@@ -31,12 +31,12 @@
 
 /* Global Message - Generic */
```

Comment:
@cujomalainey Sorry to bother a bit more, but can you update the commit message still a bit. "Implicit values may have a length of 15bits" (@lyakh 's point, it depends on platform/compiler, but may be s16) .

Also, I didn't notice earlier but "Ubsan discovered this but" is a bit out-of-place as upstream kernel. Maybe "Reported-by:" tag with Ubsan's full email...?

Reply:
isn't UBsan a tool? https://www.kernel.org/doc/html/latest/dev-tools/ubsan.html

Reply:
heh, you live and learn. I thought it was the person doing the fuzzer development :)

Reply:
me too. I googled to see what this person's GitHub ID was :-) 
Who said we can't have fun at work...